update.sh

```diff
@@ -368,6 +368,18 @@ patch__compose__add_rabbitmq() {
     fi
 }
 
+patch__compose__add_rabbitmq_hostname() {
+    if ! cat -vt "${UCRM_PATH}/docker-compose.yml" | grep -Eq "    hostname: ucrm-rabbitmq";
+    then
+        echo "Your docker-compose doesn't contain hostname in RabbitMQ section. Trying to add."
+        sed -i -e "s/image: rabbitmq:3/&\n    hostname: ucrm-rabbitmq/g" "${UCRM_PATH}/docker-compose.yml"
```

Comment:
Can this fail?

Reply:
Only if the rabbitmq section does not exist and since the check for that is right above, it should never fail.